src/interfaces/exportManager.ts

```diff
@@ -0,0 +1,21 @@
+import { FeatureCollection } from '@turf/turf';
+import { CreateExportTaskRequest, CreateExportTaskResponse, TaskParameters } from './task';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+export interface GetEstimatedSizeParams {
+  ROI?: FeatureCollection;
+  additional?: Record<string, unknown>;
+}
+
+export interface GetFileSizeParams {
+  filePath: string;
+  additional?: Record<string, unknown>;
+}
+
+export declare type GeometryMetadata = Record<string, unknown>;
+
+export interface IExportManager {
+  createExportTask: (data: CreateExportTaskRequest<TaskParameters>) => Promise<CreateExportTaskResponse>;
+  getEstimatedSize: (params?: GetEstimatedSizeParams) => Promise<number>;
+  getFileSize: (catalogRecordID: string, params: GetFileSizeParams) => Promise<number>;
```

Comment:
Consider to remove **getFileSize**.

**getEstimatedSize** I think may receive same types as **createExportTask** 

As well very nice to have an indication how space is available 
![image](https://user-images.githubusercontent.com/23166717/236813571-4afff307-85c3-4ead-9a7a-67091f46fea6.png)